alts/src/main/java/io/grpc/alts/transportsecurity/AltsAuthContext.java

```diff
@@ -17,12 +17,14 @@
 package io.grpc.alts.transportsecurity;
 
 import com.google.common.annotations.VisibleForTesting;
+import io.grpc.Internal;
 import io.grpc.alts.Altscontext.AltsContext;
 import io.grpc.alts.Handshaker.HandshakerResult;
 import io.grpc.alts.TransportSecurityCommon.RpcProtocolVersions;
 import io.grpc.alts.TransportSecurityCommon.SecurityLevel;
 
 /** AltsAuthContext contains security-related context information about an ALTs connection. */
+@Internal
 public final class AltsAuthContext {
```

Comment:
If this is actually internal, you should make the class package private, and using an Internal accessor.   See some of the classes named Internal* in the core repo.

Reply:
AltsAuthContext and TsiPeer are not used today, but will be used in the future for authorization check, thus they are public. let me update them as ExperimentalApi